app/pods/publications/publication/proofs/documents/template.hbs

```diff
@@ -79,66 +79,75 @@
-                  <Auk::Icon @name="check" @skin="success-dark" />
-                {{else}}
-                  {{t "dash"}}
-                {{/if}}
-              </td>
-              <td>
-                <WebComponents::AuButtonToolbar>
-                  <Auk::Button
-                    @skin="borderless" @layout="icon-only" @icon="download"
-                    @href={{piece.file.namedDownloadLink}}
-                    download={{await piece.downloadFilename}}>
-                    {{t "download"}}
-                  </Auk::Button>
-                  <Auk::Button
-                    @skin="borderless" @layout="icon-only" @icon="book"
-                    @href={{piece.viewDocumentURL}}
-                    target="_blank">
-                    {{t "read"}}
-                  </Auk::Button>
-                  <DropdownMenu
-                    @skin="borderless"
-                    @icon="more"
-                  as |Menu|>
-                    <Menu.Item {{on "click" (fn this.openPieceEditModal piece)}}>
-                      {{t "edit-proof-details"}}
-                    </Menu.Item>
-                  </DropdownMenu>
-                </WebComponents::AuButtonToolbar>
-              </td>
-            </tr>
+          {{#each this.pieceRows as |pieceRow|}}
+            {{#let pieceRow.piece as |piece|}}
+              <tr>
+                <td class="auk-table-checkbox-col">
+                  <Auk::Checkbox
+                    @checked={{includes pieceRow this.selectedPieceRows}}
+                    {{on "input" (fn this.togglePieceSelection pieceRow)}} />
+                </td>
+                <td>
+                  {{piece.name}}
+                </td>
+                <td>
+                  <Utils::Documents::FileTypePill @extension={{pieceRow.file.extension}} />
+                </td>
+                <td>
+                  {{#if piece.receivedDate}}
+                    {{moment-format piece.receivedDate}}
+                  {{else}}
+                    {{t "dash"}}
+                  {{/if}}
+                </td>
+                <td>
+                  <p>
+                    {{moment-format piece.created}}
+                  </p>
+                  <p class="auk-u-muted auk-u-text-small">
+                    {{moment-format piece.created "HH:mm"}}
+                  </p>
+                </td>
+                <td class="auk-u-text-align--center">
+                  {{#if piece.publicationSubcaseCorrectionFor}}
+                    <Auk::Icon @name="check" @skin="success-dark" />
+                  {{else}}
+                    {{t "dash"}}
+                  {{/if}}
+                </td>
+                <td>
+                  <WebComponents::AuButtonToolbar>
+                    <Auk::Button
+                      @skin="borderless" @layout="icon-only" @icon="download"
+                      @href={{pieceRow.file.namedDownloadLink}}
+                      download={{await piece.downloadFilename}}>
```

Comment:
Waarom de `await` hier? Kunnen we dan niet beter `pieceRow.file.downloadFilename` gebruiken?